_DeploymentAndDistroPackaging/ansible/roles/keystone/templates/openrc.j2

```diff
@@ -7,6 +7,6 @@ export OS_TENANT_NAME=admin
 export OS_USERNAME=admin
 export OS_PASSWORD={{ keystone_admin_password }}
 export OS_AUTH_URL=https://{{ keystone_fqdn }}:35357/v3
-export OS_CACERT=certificates/keystone/keystone_cert.pem
+export OS_CACERT={{ workdir }}/keystone_cert.pem
```

Comment:
We are installing the certificate on the controller node only.

This file is created on the deployment node and allows the user to run the cli on any machine without need to install the certificate